### PR TITLE
Globalize hazard

### PR DESCRIPTION
**Pull request by meomancer**
this fix https://trello.com/c/I4eOwjvk/184-fba-support-other-types-of-hazards

- globalize hazard references (previously, named flood)

### CODE REVIEW

File: fixtures/data/00_references/11_hazard_class.sql [hunk 1]
```diff
@@ -1,4 +1,12 @@
-INSERT INTO public.hazard_class (id, min_m, max_m, label) VALUES (1, 0, 0.7, '0 - 0.7m') ON CONFLICT DO NOTHING ;
-INSERT INTO public.hazard_class (id, min_m, max_m, label) VALUES (2, 0.7, 1.5, '0.7 - 1.5m') ON CONFLICT DO NOTHING ;
-INSERT INTO public.hazard_class (id, min_m, max_m, label) VALUES (3, 1.5, 3, '1.5 - 3.0m') ON CONFLICT DO NOTHING ;
-INSERT INTO public.hazard_class (id, min_m, max_m, label) VALUES (4, 3, 999999, '3 or greater') ON CONFLICT DO NOTHING ;
+-- hazard CLASS --
```
Comment (lucernae) [inline fixtures/data/00_references/11_hazard_class.sql]: Separate GeoCRIS specific hazard/schema in a different module.
Leave the original FbF shema be.

File: documents/backend-development/backend-python-unit-test.md [hunk 2]
```diff
@@ -48,9 +48,9 @@ python3 -m unittest tests.backend.test_building_impact -v
 # run tests in `TestBuildingImpact` class in `tests.backend.test_building_impact` module
 python3 -m unittest tests.backend.test_building_impact.TestBuildingImpact -v
 
-# run tests for method name `test_non_flooded_building_summary` in a 
+# run tests for method name `test_non_hazarded_building_summary` in a
```
Comment (lucernae) [inline documents/backend-development/backend-python-unit-test.md]: Let's have a convention for `non_flooded` or `non_hazarded` phrase, we just omit it.
So `non_hazarded_building_summary` becomes simply `building_summary`.
It makes it easy to understand and search.

File: fixtures/data/00_references/10_hazard_type.sql [hunk 3]
```diff
@@ -1 +1,2 @@
 INSERT INTO public.hazard_type (id, name) VALUES (1, 'Flood') ON CONFLICT DO NOTHING ;
+INSERT INTO public.hazard_type (id, name) VALUES (2, 'Hurricane') ON CONFLICT DO NOTHING ;
```
Comment (lucernae) [inline fixtures/data/00_references/10_hazard_type.sql]: GeoCRIS specific definitions needs to be on separate module.
We need to add new Flood classification for GeoCRIS too.

File: fixtures/schema/02_fbf/00_tables/00_references/05_hazard_class.sql [hunk 4]
```diff
@@ -29,10 +28,8 @@ CREATE SEQUENCE IF NOT EXISTS public.depth_class_id_seq
...
 ALTER SEQUENCE public.depth_class_id_seq OWNED BY public.hazard_class.id;
 
-
-
 --
 -- Name: hazard_class id; Type: DEFAULT; Schema: public; Owner: -
 --
 
-ALTER TABLE ONLY public.hazard_class ALTER COLUMN id SET DEFAULT nextval('public.depth_class_id_seq'::regclass);
+ALTER TABLE ONLY public.hazard_class ALTER COLUMN id SET DEFAULT nextval('public.depth_class_id_seq'::regclass);
```
Comment (lucernae) [inline fixtures/schema/02_fbf/00_tables/00_references/05_hazard_class.sql]: change the sequence name too.

**Review by lucernae — COMMENTED** [review]
@meomancer I'm sorry, there are too many files to review.

Let's break this up into more manageable chunk of commits.

First, let's agree with the following conventions:

1. Code refactor for main FbF schema and schema for GeoCRIS only should be in a different module/commits
2. Schema change and data change should be in a separate commit
3. Integration test and unit test refactor commit should be the last
4. You are allowed to force push into this PR/branch
5. With this huge refactor we have to focus on making the test pass first, then generate schema migrations

**Comment by meomancer** [comment]
hi @lucernae 
i have changed everything
- make commits more modular
- change flood to hazard
- change non_flooded name into just mv_building_summary (etc)
- check test

also we have issue, which is we need to update smart excell
because we rename everything, we need a new smart excell function (with proper table name)
currently we are using **FbfFloodData** as the model name, and this PR change to **FbfHazardData**
as well as every table name that in that model

it makes every test based on the smart excell is broken, so we need to update smart excell first

**Pull request closed without merging**